frontend/src/data/config/DatasetMetadata.ts

```diff
@@ -937,52 +941,62 @@ export const DatasetMetadataMap: Record<DatasetId, DatasetMetadata> = {
   'cdc_wisqars_youth_data-youth_by_race_and_ethnicity_state_current': {
     name: 'Gun-related Youth and Young Adult Deaths, by race and ethnicity and state',
     original_data_sourced: '2021',
+    contains_nh: true,
     source_id: 'cdc_wisqars_data'
   },
   'cdc_wisqars_youth_data-youth_by_race_and_ethnicity_state_historical': {
     name: 'Gun-related Youth and Young Adult Deaths, by race and ethnicity and state',
     original_data_sourced: '2018-2021',
+    contains_nh: true,
     source_id: 'cdc_wisqars_data'
   },
   'cdc_wisqars_black_men_data-black_men_by_urbanicity_national_current': {
     name: 'Gun homicides for Black men, by urbanicity (e.g. Metro or Non-Metro), nationally',
     original_data_sourced: '2021',
+    contains_nh: true,
     source_id: 'cdc_wisqars_data'
   },
   'cdc_wisqars_black_men_data-black_men_by_urbanicity_national_historical': {
     name: 'Gun homicides for Black men, by urbanicity (e.g. Metro or Non-Metro), nationally',
     original_data_sourced: '2018-2021',
+    contains_nh: true,
     source_id: 'cdc_wisqars_data'
   },
   'cdc_wisqars_black_men_data-black_men_by_urbanicity_state_current': {
     name: 'Gun homicides for Black men, by urbanicity (e.g. Metro or Non-Metro) and state',
     original_data_sourced: '2021',
+    contains_nh: true,
     source_id: 'cdc_wisqars_data'
   },
   'cdc_wisqars_black_men_data-black_men_by_urbanicity_state_historical': {
     name: 'Gun homicides for Black men, by urbanicity (e.g. Metro or Non-Metro) and state',
     original_data_sourced: '2018-2021',
+    contains_nh: true,
     source_id: 'cdc_wisqars_data'
   },
 
   'cdc_wisqars_black_men_data-black_men_by_age_national_current': {
     name: 'Gun homicides for Black men, by age, nationally',
     original_data_sourced: '2021',
+    contains_nh: true,
     source_id: 'cdc_wisqars_data'
   },
   'cdc_wisqars_black_men_data-black_men_by_age_national_historical': {
     name: 'Gun homicides for Black men, by age, nationally',
     original_data_sourced: '2018-2021',
+    contains_nh: true,
     source_id: 'cdc_wisqars_data'
   },
   'cdc_wisqars_black_men_data-black_men_by_age_state_current': {
     name: 'Gun homicides for Black men, by age and state',
     original_data_sourced: '2021',
+    contains_nh: true,
     source_id: 'cdc_wisqars_data'
   },
   'cdc_wisqars_black_men_data-black_men_by_age_state_historical': {
```

Comment:
should we also update the definition in the metric config to reflect that the "black men" topics are for Black NH?